common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -41,16 +41,138 @@ Features
 Configuration
 -------------
 
-To enable MSP OSD, set the following parameters ( example using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
+To enable telemetry based MSP OSD, set the following parameters ( example using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
 
  - :ref:`OSD_TYPE<OSD_TYPE>` = 3 if no integrated OSD is being used in order to activate the OSD code. If an integrated OSD is present and the user wishes to have both OSDs , then :ref:`OSD_TYPE<OSD_TYPE>` = 1 will activate the on-board OSD as well as providing screens for the MSP OSD function. For example, on vehicles using the DJI goggles/air system for medium range, but still running a long range VTX using the internal OSD for when the vehicle exceeds the range of the HD DJI Goggles. This configuration could use one OSD screen optimized for DJI Goggles, and another for the integrated OSD and the user can switch between them depending on which video system is being viewed.
  - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 33
- - :ref:`MSP_OPTIONS<MSP_OPTIONS>` = 0 (polling mode)
+ - :ref:`SERIAL2_BAUD<SERIAL2_BAUD>` = 115
+ - :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 0 = 1 (EnableTelemetryMode)
  
 .. note:: Serial port buad rate default is changed to 115.2Kbaud automatically when setting the above protocol type. However, if the user has previously or subsequently changes the baud, this default will not be used. 115.2Kbaud is required by most video goggle systems.
 
 .. note:: DJI Custom OSD must be enabled: in SETTINGS->DISPLAY->CUSTOM OSD menu of goggles
 
+DisplayPort OSD
```

Comment:
Moving all of this before the panel items section is wrong...these are MSP only panel items...DisplayPort and integrated OSDs display MANY more panels (all of them that are not MSP only)....

Reply:
Hmm -It is VERY confusing to have all these OSD values listed on the page and have to scroll past all this stuff to get to the DisplayPort part. A couple of options, what do you think:
1. Put DisplayPort first - then the OSD list can be at the bottom of the page and only those with a telemetry OSD have to scroll there to see it
2. Split this into two pages.

Reply:
Thanks @Hwurzburg - I'll try that.